libc/src/poll/linux/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+add_entrypoint_object(
+  poll
+  SRCS
+    poll.cpp
+  HDRS
+    ../poll.h
+  DEPENDS
+    libc.hdr.types.nfds_t
+    libc.hdr.types.struct_pollfd
+    libc.hdr.types.struct_timespec
+    libc.include.poll
```

Comment:
This is not correct. Instead, I should put this line exactly in libc/config/linux/{arch}/headers.txt.

via https://github.com/llvm/llvm-project/issues/125102#issuecomment-2625933083

Reply:
6fc7519871b6